plugins/services/src/js/reducers/serviceForm/FormReducers/LocalVolumes.js

```diff
@@ -0,0 +1,52 @@
+import {
+  ADD_ITEM,
+  REMOVE_ITEM,
+  SET
+} from '../../../../../../../src/js/constants/TransactionTypes';
+import {parseIntValue} from '../../../../../../../src/js/utils/ReducerUtil';
+
+module.exports = {
+  FormReducer(state = [], {type, path, value}) {
+    if (path == null) {
+      return state;
+    }
+
+    const joinedPath = path.join('.');
+
+    if (joinedPath.search('localVolumes') !== -1) {
+      if (joinedPath === 'localVolumes') {
+        switch (type) {
+          case ADD_ITEM:
+            state.push({containerPath: null, size: null, mode: 'RW'});
+            break;
+          case REMOVE_ITEM:
+            state = state.filter((item, index) => {
+              return index !== value;
+            });
+            break;
+        }
+
+        return state;
+      }
+
+      const index = joinedPath.match(/\d+/)[0];
```

Comment:
I am okay with the change request but wouldn't it be better to change this in all places where this pattern has been used instead of only for this reducer? However I will change this in this PR for all the places where this pattern has been used in this PR and create an issue for the rest.